monitoring/rules/viya/rules-sas-launcher-job.yaml

```diff
@@ -9,24 +9,31 @@ spec:
   - name: sas-launcher-job-info.rules
     interval: 6s
     rules:
-    - expr: sum((label_replace(kube_pod_labels{pod=~"sas-.+"},"job_pod","$1","pod","(.+)") * on (job_pod) group_left(job_name,queue_name) sas_job_info{job_pod=~"sas-.+"}) * on (pod) group_left(node) kube_pod_info{pod=~"sas-.+"}) without (container, endpoint, instance, job, service)
+    - expr: sum((label_replace(kube_pod_labels{label_sas_com_created_by="sas-launcher"},"job_pod","$1","pod","(.+)") * on (job_pod) group_left(job_name,queue_name) sas_job_info{job_pod=~"sas-.+"}) * on (pod) group_left(node) kube_pod_info{pod=~"sas-.+"}) without (container, endpoint, instance, job, service)
       record: ':sas_launcher_pod_info:'
     - expr: sum(:sas_launcher_pod_info:* on (namespace, pod) group_left(phase) (kube_pod_status_phase{pod=~"sas-.+"}==1)) without (endpoint,instance,job,service)
       record: ':sas_launcher_pod_status:'
   - name: sas-launcher-job-metrics.rules
     interval: 10s
     rules:
-    - expr: :sas_launcher_pod_status:{phase="Running"} * on (namespace,pod) group_left sum(rate(container_cpu_usage_seconds_total{container="",pod=~"sas-.+",cpu="total"}[1m])) by (namespace,pod)
+    # - expr: :sas_launcher_pod_status:{phase="Running"} * on (namespace,pod) group_left sum(rate(container_cpu_usage_seconds_total{container="",pod=~"sas-.+",cpu="total"}[1m])) by (namespace,pod)
+    - expr: :sas_launcher_pod_status:{phase="Running"} * on (namespace,pod) group_left sum(rate(container_cpu_usage_seconds_total{container="",cpu="total"}[1m])) by (namespace,pod)
```

Comment:
It's interesting that we do not need to limit/filter the containers to just those from specific SAS pods anymore.